ecs-agent/tmds/handlers/v4/state/response.go

```diff
@@ -38,7 +38,7 @@ type TaskResponse struct {
 	EphemeralStorageMetrics *EphemeralStorageMetrics `json:"EphemeralStorageMetrics,omitempty"`
 	CredentialsID           string                   `json:"-"`
 	TaskNetworkConfig       *TaskNetworkConfig       `json:"-"`
-	FaultInjectionEnabled   bool                     `json:"-"`
```

Comment:
I think the current name is intentional.

Reply:
+1

Reply:
we have a change for this field in the feature branch

Reply:
Perhaps we should add a comment either in the PR description or in the code denoting that `EnabledFaultInjection` within the Task struct of `agent/api/task` essentially relates/maps to `FaultInjectionEnabled` within the task response struct of `ecs-agent/tmds/handlers/v4/state/`